scripting/2.1/modules/ROOT/pages/scripting-module-upgrade-migrate.adoc

```diff
@@ -62,6 +62,20 @@ Browse the engine dependency and install it. +
 ** `<sharedLibraries>` tags under the `<configuration>` section
 ** `<dependency>` tags under the `<dependencies>` section
 
+[NOTE]
+Starting with Java 17, for your script to execute JavaScript code, it must use GraalVm engine. Therefore, you must add in your `pom.xml` file the following library dependencies: +
```

Comment:
```suggestion
Starting with Java 17, execution of JavaScript code requires the GraalVm engine. To use this engine, add the following library dependencies to your `pom.xml` file : +
```
Take it or leave it. I'm trying to avoid "it must use".